### PR TITLE
carloshz4/c02-network02

### PR DESCRIPTION
**Pull request by carloshz4**
# Description

Exercise: c02-network02

Student Name: Carlos Zambrano

> Added README.md inside folder  /classes/02class/exercises/c02-network02/carloshz4/

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/02class/exercises/c02-network02/carloshz4/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|
+|-|-|-|-|-|-|
+|subnet-a|10.0.0.0/18|10.0.0.1-10.0.63.254|16382|10.0.63.255|16379
+|subnet-b|10.0.64.0/18|10.0.64.1-10.0.127.254|16382|10.0.127.255|16379
+|subnet-c|10.0.128.0/18|10.0.128.1-10.0.191-254|16382|10.0.191.255|16379
```
Comment (roperto) [inline classes/02class/exercises/c02-network02/carloshz4/README.md:11]: Small typo here on the IP `10.0.191-254`

File: classes/02class/exercises/c02-network02/carloshz4/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# c02-network02
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|
```
Comment (roperto) [inline classes/02class/exercises/c02-network02/carloshz4/README.md:7]: I think the AWS Reserved is the other way around -- this is the number of non-reserved.

File: classes/02class/exercises/c02-network02/carloshz4/README.md (hunk 1 above)
Comment (carloshz4) [reply classes/02class/exercises/c02-network02/carloshz4/README.md:11]: Thanks Daniel, I'll fix it.

File: classes/02class/exercises/c02-network02/carloshz4/README.md (hunk 2 above)
Comment (carloshz4) [reply classes/02class/exercises/c02-network02/carloshz4/README.md:7]: Fair enough, I thought that'd be always the same value. As I understand it'll be 5 IP's (including network IP, GW, .2 and .3 used by AWS, and the broadcast one).  So, would you say it's 5 all the time?

**Comment by carloshz4** [comment]
Hey @roperto , sorry for the delay, I've updated this PR with the requested changes.

**Pull request merged**